Update scope guide with access section (#141)
meilisearch/integration-guides: guides/scopes.md

| [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) | [bidoubiwa](https://github.com/bidoubiwa/) |
| [docs-searchbar.js](https://github.com/meilisearch/docs-searchbar.js) | [bidoubiwa](https://github.com/bidoubiwa/) |
|||

> | [Firestore extension](https://github.com/meilisearch/firestore-meilisearch) (WIP) | [bidoubiwa](https://github.com/bidoubiwa/) |

Reviewer: Is this not the scope of @alallema for the moment?

Author: Yes for the moment, but it will finally be your scope as we discussed (since it's JS).
Would you rather I put amelie until you get the repo?

Reviewer: No its oke I was just wondering

| [Cloud Scripts](https://github.com/meilisearch/cloud-scripts) | [alallema](https://github.com/alallema/) |
|||
| [docs-scraper](https://github.com/meilisearch/docs-scraper)| [bidoubiwa](https://github.com/bidoubiwa/) |
| [js-project-boilerplate](https://github.com/meilisearch/js-project-boilerplate)| [bidoubiwa](https://github.com/bidoubiwa/) |
|||
| [WordPress integration](https://github.com/meilisearch/docs-scraper) (Stopped)| [curquiza](https://github.com/curquiza/) |

### Access

> When a member of the integration team owns a repository, he/she has full access to this repository, i.e. Admin access on GitHub.

Reviewer: ```suggestion
When a member of the integration team owns a repository, they have full access to this repository, i.e. Admin access on GitHub.
```

They is the neutral form even in singular circumstances

> However, a great power comes with a great responsibility. It means:

Reviewer: ```suggestion
However, a great power comes with great responsibilities. It means:
```

Thats the original sentences and sinces there are multiple bullet points it is more accurate?

Author: 😇 
https://www.google.com/search?channel=trow5&client=firefox-b-d&q=a+great+power+comes+with+a+great+responsibility

Reviewer 2: Why don't we use the real sentence in full:
`However, with great power comes great responsibility`

> - Merging with Bors (using `bors merge`) should always be privileged. Only merge manually they is no other alternative.

Reviewer: ```suggestion
- Merging with Bors (using `bors merge`) should always be privileged. Only merge manually if there is no other alternative.
```
